Doc/library/http.cookies.rst

```diff
@@ -137,11 +137,19 @@ Morsel Objects
    * ``secure``
    * ``version``
    * ``httponly``
+   * ``samesite``
 
    The attribute :attr:`httponly` specifies that the cookie is only transferred
    in HTTP requests, and is not accessible through JavaScript. This is intended
    to mitigate some forms of cross-site scripting.
 
+   .. versionchanged:: 3.7
```

Comment:
@akash0x53 I'm really sorry that I wasn't being clear about this :(

The `versionchanged` info is to be added after the paragraph about `samesite` attribute.

Overall the change should look like this:

```
   ...
   to mitigate some forms of cross-site scripting.

   The attribute :attr:`samesite` specifies that browser is not allowed to send the
   cookie along with cross-site requests. This help to mitigate CSRF attacks. Valid
   values for this attribute are ``'Strict'`` and ``'Lax'``.

   .. versionchanged:: 3.7
      Added support for :attr:`samesite` attribute.

   The keys are case-insensitive and their default value is ``''``.
```